_migration-assistant/migration-phases/migrate-metadata/transform-string-text-keyword.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: default
+title: Transform string to text/keyword
+nav_order: 4
+parent: Migrate metadata
+grand_parent: Migration phases
+permalink: /migration-assistant/migration-phases/migrate-metadata/transform-string-text-keyword/
+---
+
+# Transform string to text/keyword fields
+
+
+This guide explains how the Migration Assistant automatically handles the deprecated `string` field type during migration from older Elasticsearch versions.
+
+## Overview
+
+The `string` field type was the primary text field type in early versions of Elasticsearch but was deprecated in Elasticsearch 5.x and completely removed in Elasticsearch 6.0. In Elasticsearch 5.x, it remained available in backward compatibility mode only.
+
+When migrating from Elasticsearch 1.x through 5.x to newer versions, the Migration Assistant automatically converts `string` field types to their modern equivalents: `text` for analyzed fields and `keyword` for non-analyzed fields.
+
+To determine if an Elasticsearch cluster uses `string` field types, make a call to your source cluster's `GET /_mapping` api. On the Migration Console, run `console clusters curl source_cluster "/_mapping"`.  If you see `"type":"string"`, then this transformation is applicable and these fields will be automatically transformed during migration.
+
+## Compatibility
+
+The `string` to `text`/`keyword` field type transformation applies to:
+- **Source clusters**: Elasticsearch 1.x - 5.x
+- **Target clusters**: Elasticsearch 5.x+ or OpenSearch 1.x+
+- **Automatic conversion**: No configuration required during metadata
+
+## Automatic conversion logic
+
+The Migration Assistant determines whether to convert a `string` field to `text` or `keyword` based on the field's `index` property.
+
+### Conversion to `keyword`
+Fields are converted to `keyword` when:
+- `index: "not_analyzed"`
+- `index: "no"`
+- `index: false`
+
+### Conversion to `text`
+Fields are converted to `text` when:
+- `index: "analyzed"` (default behavior)
+- `index` property is not specified
+- Any other `index` value
```

Comment:
This line should be formatted in the same way as the preceding two. This also appears to be an incomplete phrase.